Lab-2.ipynb

```diff
@@ -0,0 +1,623 @@
+      "text/plain": [
+       "<Figure size 2016x504 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plotter(\"Poisson\",p3,3,\"sum\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Above, I wrote two functions: one to convolve a specified distribution *N* number of times, and another to plot either a discrete or continuous graph based on those results.\n",
+    "\n",
+    "Given in the problem setup, the average cosmic ray background noise follows a discrete probablity mass function, the Poisson distribution. I have arbitrarily selected X = 2 for a mean count of cosmic ray noise per day. Then, using my convolve function, I summed this Poisson background for 3 days using the convolution operator 3 times. The graphs above show the resultant distribution for the first, second, and third convolution.\n",
+    "\n",
+    "We can see that the average cosmic ray count will shift up on the x-axis, meaning as we sum over multiple (here 3) days, we will find more cosmic ray counts over that time period. This operation simply sums up the total average cosmic ray counts over a specified number of days, if on day 1 we know there is an average of 2 counts. The probabilities decrease as we sum up for more days, and the width of the distribution is slightly wider each following day. This indicates we are less likely to measure a larger cosmic ray count over more days, and that there are more possible average values to measure as well.\n",
+    "\n",
+    "As we sum for more days, the distribution retains a Poisson shape. This is evident especially in the semilog plot comparison, where the logs of the Poisson plots do not approach a parabolic shape (which would indicate a Gaussian shape)."
```

Comment:
1.A: my --> the. if --> an example being on day 1. Be more specific, what probabilities decrease, you mean the probability for locating each specific average cosmic ray value decreases. Graphs look good.